arangod/Cluster/MaintenanceFeature.cpp

```diff
@@ -747,3 +747,33 @@ arangodb::Result MaintenanceFeature::copyAllErrors(errors_t& errors) const {
 }
 
 
+uint64_t MaintenanceFeature::shardVersion (std::string const& shname) const {
+  MUTEX_LOCKER(guard, _versionLock);
+  auto const it = _shardVersion.find(shname);
+  LOG_TOPIC(TRACE, Logger::MAINTENANCE)
+    << "getting shard version for '"  << shname << "' from " << _shardVersion;
+  return (it != _shardVersion.end()) ? it->second : 0;
+}
+
+
+void MaintenanceFeature::incShardVersion (std::string const& shname) {
```

Comment:
This should return the resulting shard version as a uint64_t.

Reply:
Fine. Did originally consider decided against. But if it makes you happy :)